### DISCUSSION TITLE
Richtlijnen: Goede labelteksten voor formulieren

### DISCUSSION THREAD

**Discussion by YHoekstra** [nl-design-system/documentatie#475] [Richtlijnen NLDS]
Vaak wordt in verschillende formulieren verschillende woorden gebruikt voor zelfde inputfield. 
Voorbeeld: 
- Persoonlijke gegevens / Uw gegevens /
- Achternaam / Achternaam waarmee u geboren bent / Uw naam
- Voorletter(s) / Voorletters / Voorna(a)m(en)
- BSN-nummer / burgerservicenummer /nummer identiteitsbewijs

Kunnen we gezamenlijk een woordenboek aanleggen zodat alle gemeente zelfde woorden gaan gebruiken?

**Comment by rianrietveld** [comment]
@YHoekstra 
Interessante vraag! We willen een lijst van teksten voor foutmeldingen gaan aanleggen, een lijst van labels is misschien ook een idee. Ik ga het in de groep gooien.
